lib/src/license_detection/crc32.dart

```diff
@@ -0,0 +1,277 @@
+//CRC specification
+// Width  : 32
+// Poly   : 0x04c11db7
+// Init   : 0xffffffff
+// RefIn  : true
+// RefOut : true
+// Decimal valyue type: Unsigned
+// CRC 32 Normal specification: https://en.wikipedia.org/wiki/Cyclic_redundancy_check
+int crc32(List<int> bytes) {
```

Comment:
Documentation is `/// ` followed by markdown.

Typically something like:
```dart
/// <summary>
///
/// <longer description>
int crc32(List<int> bytes) {
```

For example could be:
```dart
/// Compute CRC-32 for [bytes], returns an unsigned 32 bit integer.
///
/// This is same [CRC-32][1] as is used in IEEE 802.3 Ethernet, `gzip`, `cksum` and others.
///
/// [1]: https://en.wikipedia.org/wiki/Cyclic_redundancy_check
int crc32(List<int> bytes) {
```